app/models/sushi.rb

```diff
@@ -195,19 +183,22 @@ def coerce_metric_types(params = {})
   module QueryParameterValidation
     extend ActiveSupport::Concern
     included do
-      attr_reader :item_id, :platform
+      attr_reader :item_id, :item_id_in_params
+      attr_reader :platform, :platform_in_params
```

Comment:
Nitpick (non-blocking): personally, this reads easier for me: 

```suggestion
      attr_reader :item_id, :item_id_in_params, :platform, :platform_in_params
```

Is there a reason to list them separately?

Reply:
I debated listing it the way you have it. I wondered if it may read easier to have the attributes grouped by likeness instead? I have more filters to do so I may change this in the future.